### PR TITLE
Pm 2021 12 styling issues jan milestone

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

File: meinberlin/apps/budgeting/assets/FilterBar.jsx [hunk 1]
```diff
@@ -37,43 +37,32 @@ export const FilterBar = props => {
   }
 
   const prepareFilter = (filter, filterType) => {
```
Comment (phillimorland) [inline meinberlin/apps/budgeting/assets/FilterBar.jsx:39]: > :last-child {
            float: right;
            margin-right: 0;
        }

**Review by khamui — APPROVED** [review]
looking good to me!

File: meinberlin/apps/budgeting/assets/FilterBar.jsx (hunk 1 above)
Comment (Rineee) [reply meinberlin/apps/budgeting/assets/FilterBar.jsx:39]: I guess a question to @khamui ?

**Review by Rineee — APPROVED** [review]
Yeah, cool!

**Pull request merged**